styx-client/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java

```diff
@@ -340,16 +360,16 @@ public void testTokenSuccess() {
   }
 
   @Test
-  public void testTokenFailure() throws Exception {
+  public void testTokenFailure() throws IOException, java.security.GeneralSecurityException {
     final IOException rootCause = new IOException("netsplit!");
     when(auth.getToken(any())).thenThrow(rootCause);
     final StyxApolloClient styx = new StyxApolloClient(client, CLIENT_HOST, auth);
     final CompletableFuture<Void> f = styx.triggerWorkflowInstance("foo", "bar", "baz")
         .toCompletableFuture();
     try {
-      f.get();
+      f.join();
```

Comment:
Why the change here?

Reply:
unchecked exception, nicer to handle than `ExecutionException` etc

Reply:
You're right, in this context it doesn't help much. I was too eager :)